CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ add_app_subdirectory(${ROVER_APPS_DIR}/science_2021)
 # Include Test Apps
 add_app_subdirectory(${TEST_APPS_DIR}/test-actuator-controller)
 add_app_subdirectory(${TEST_APPS_DIR}/test-adafruitSTEMMA)
-add_app_subdirectory(${TEST_APPS_DIR}/test-aeat-6012)
+add_app_subdirectory(${TEST_APPS_DIR}/test-aeat-6012)\
```

Comment:
```suggestion
add_app_subdirectory(${TEST_APPS_DIR}/test-aeat-6012)
```

Reply:
Fixed in 0fabe3f72fab676fa8c9a4c6ba2574a79b1cc32f